readme.md

```diff
@@ -4,6 +4,8 @@
 
 Additional description about the project and its features.
 -Phase 1. No features added yet.
+-Phase 2. Added CRUD features.
+-Phase 3. Added the functionality to tick mark completed tasks. Also added the functionality to remove all completed tasks.
 
 # Getting Started
 
```

Comment:
- [OPTIONAL] Please it will be nice to add explicit steps to clone and run your project locally, as this will help none technical users, and some of these are recruiters, to clone and run your project locally in order to see how it works.

Reply:
I have already updated readme file for the same.